governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Should we add a new WBTC vault type with lower LR? - January 11, 2021
+summary: This poll will provide guidance to the Domain Teams for creating a new WBTC based vault type
```

Comment:
I think I would take this straight from the greenlight polls: 'Signal your support or opposition to prioritising onboarding a WBTC-B vault-type'